### PR TITLE
Update some demos

### PR DESCRIPTION
**Pull request by micheles**
Provided by Catalina. The new demos are running here:
https://ci.openquake.org/job/zdevel_oq-engine/1488/console
https://ci.openquake.org/job/zdevel_oq-risklib/1137/

### CODE REVIEW

**Comment by CatalinaYepes** [comment]
All the new files look good to me.
However:
- The `structural_fragility_model.xml` should be in a tidy format
- I don't see the changes for edRisk, ClassicalRisk and ClassicalBCR

**Comment by daniviga** [comment]
There's a mix of files with and without newline at the end. Maybe we homogenize all the files to have a newline at the end.

File: demos/EventBasedRisk/job.ini [hunk 1]
```diff
@@ -1,65 +1,54 @@
...
+specific_assets = 
 
-export_dir = /tmp
+[outputs]
 ground_motion_fields = false
-hazard_curves_from_gmfs = true
-mean_hazard_curves = true
+hazard_curves_from_gmfs = false
+mean_hazard_curves = false
 quantile_hazard_curves =
```
Comment (micheles) [inline demos/EventBasedRisk/job.ini]: Catalina, notice that in the document about the outputs it is marked as critical the fact that "Event based demo is not exporting the maps and curves". However with `hazard_curves_from_gmfs = false` curves should not be generated nor exported. Are you sure the flag must be false? Also, if you want to generate maps you must add the appropriate configuration parameter.

**Comment by micheles** [comment]
@daniviga please notice that the names of the directories have changed (i.e. ProbabilisticEventBased -> EventBasedRisk), is this a problem for you?

**Comment by daniviga** [comment]
@micheles no problem on my side

File: demos/EventBasedRisk/job.ini (hunk 1 above)
Comment (CatalinaYepes) [inline demos/EventBasedRisk/job.ini]: Michele, better if we change it to true. The idea is to generate at least one map.

**Comment by CatalinaYepes** [comment]
@micheles, can we split the demos in two folders (hazard_demos and risk_demos, as before)?

**Comment by micheles** [comment]
Done, the demos are now split in two folders: demos/hazard and demos/risk

**Comment by CatalinaYepes** [comment]
LGTM

**Comment by daniviga** [comment]
LGTM for me too. We'll need to update documentation on the engine.

**Pull request merged**